README.md

```diff
@@ -1,4 +1,48 @@
-CraftedWeb
-==========
+# CraftedWeb
 
-A CMS for game servers 
\ No newline at end of file
+CraftedWeb is a CMS for World of Warcraft Servers.
+
+## Features
+
+CraftedWeb has a plethora of features, outlined below.
+
+### Expansions
+|Expansion			| Supported?                    | Planned?                   |
+|------------------|-------------------------------|----------------------------|
+|Vanilla           |`No`            				| `Yes`          				|
+|Burning Crusade   |`No`            				| `Yes`         					|
+|WoTLK             |`Yes`							|							|
+|Cataclysm         |`No`							|`Yes`							|
+|Mists of Pandaria |`No`							|	`Yes`						|
+|WoD               |`No`							|`No`						|
+|Legion            |`No`							|		`Yes`					|
+
+### Main Features
+| Feature				| Supported	| Planned	|
+|-----------------------|-----------|-----------|
+| Account Management	|`Yes` 		|			|
+| Votes & Donations		|`Yes`		|			|
+| Character Tools		|`Yes`		|			|
+| Tooltip System		|`Yes`		|			|
+| Realm Status			|`Yes`		|			|
+| Multi-Realm Support	|`Yes`		|			|
+| Admin Panel			|`No*`		|`Yes`		|
+| Staff Panel			|`No*`		|`Yes`		|
+| Notification System	|`Yes`		|			|
+| News System w/ Comments|`Yes`		|			|
+| Template System		|`Yes`		|			|
+| Plugin Support		|`Yes`		|			|
+| WoWHead Integration	|`Yes`		|			|
+| OpenWoW Integration	|`Yes`		|			|
+| phpBB Integration		|`Yes**`	|			|
+
+> \* Feature exists, is not up-to-date
+>
+> \** phpBB integration may be difficult to configure for beginners - we do not offer support for this.
+
+## How does CraftedWeb Look?
+![Account Registration](https://i.imgur.com/XlPfzAo.jpg)
+
+![Vote Shop, tooltips, and account management](https://i.imgur.com/dakZszR.jpg)
+
+![Admin Panel (Coming Soon!)](https://i.imgur.com/SuxVotP.jpg)
```

Comment:
Image outdated and it should be just `Admin Panel` because it already works, not fully, but it works.

Reply:
On the images try to install on your own and provide your own images. Those are outdated.